### PR TITLE
[IAM-5212] fix docs for the new Okta SAML app according to Okta requirements

### PR DESCRIPTION
**Pull request by przemek-sl**
I based the tutorial on adding the Sauce Labs SAML app from the Okta catalog on the following okta docs:
* https://help.okta.com/en-us/Content/Topics/Apps/apps-add-applications.htm
* https://help.okta.com/en-us/Content/Topics/Apps/Apps_Overview_of_Managing_Apps_and_SSO.htm

I removed screenshots as they were taken from the old app and we do not have access to the new SAML app since it has not been yet published. Once the app is published, I will add the screenshots.

### CODE REVIEW

**Review by budziam — APPROVED** [review]

File: docs/basics/sso/configuring-sso-in-okta.md [hunk 1]
```diff
@@ -25,14 +25,11 @@ Complete the following steps to set up SAML SSO integration between Okta and Sau
 3. Click **Add Integration**.
    <img src={useBaseUrl('img/basics/sso/idp-config/okta/add-integration.png')} alt="Add Integration" width="700" />
 
-4. Leave the default name or rename it and click **Next**.
-   <img src={useBaseUrl('img/basics/sso/idp-config/okta/general-settings.png')} alt="General Settings" width="800" />
+4. Enter the required information under **General Settings**, and then click **Next**.
 
-5. Click the link **Identity Provider metadata**. The metadata will open in a new tab. Copy it and save as an XML file. Use this metadata file to set up the SSO integration at Sauce Labs in [Team Management Panel](/basics/sso/setting-up-sso#integrating-with-sauce-labs-service-provider).
-   <img src={useBaseUrl('img/basics/sso/idp-config/okta/download-metadata.png')} alt="Download Metadata" width="600" />
+5. On the **Sign On Options** page, in the **Metadata details** you find Identity Provider Metadata URL. Open it in a new tab. Copy the content and save as an XML file. Use this metadata file to set up the SSO integration at Sauce Labs in [Team Management Panel](/basics/sso/setting-up-sso#integrating-with-sauce-labs-service-provider).
```
Comment (aoleszyn) [inline docs/basics/sso/configuring-sso-in-okta.md:30]:
```suggestion
5. On the **Sign On Options** page, in the **Metadata details** section you can find the Identity Provider Metadata URL. Open it in a new tab, copy the content, and save it as an XML file. You can use this metadata file to set up the SSO Integration at Sauce Labs in the [Team Management Panel](/basics/sso/setting-up-sso#integrating-with-sauce-labs-service-provider).
```

File: docs/basics/sso/configuring-sso-in-okta.md [hunk 2]
```diff
@@ -25,14 +25,11 @@ Complete the following steps to set up SAML SSO integration between Okta and Sau
...
 
-6. In the **Credentials Details** section make sure that the value **Application username format** is a [valid email address](/basics/sso/setting-up-sso/#name-id). The default format for **Okta username** is email address, unless it was changed in Okta Admin Console.
-   <img src={useBaseUrl('img/basics/sso/idp-config/okta/app-username-format.png')} alt="Application Username Format" width="650" />
+6. On the **Sign On Options** page in Okta, in the **Credentials Details** section make sure that the value **Application username format** is a [valid email address](/basics/sso/setting-up-sso/#name-id). The default format for **Okta username** is email address, unless it was changed in Okta Admin Console.
```
Comment (aoleszyn) [inline docs/basics/sso/configuring-sso-in-okta.md:32]:
```suggestion
6. On the **Sign On Options** page in Okta, in the **Credentials Details** section, ensure that the value for **Application username format** is a [valid email address](/basics/sso/setting-up-sso/#name-id). The default format for **Okta username** is an email address, unless it has been changed in Okta Admin Console.
```

Comment (przemek-sl) [reply docs/basics/sso/configuring-sso-in-okta.md]: fixed

File: docs/basics/sso/configuring-sso-in-okta.md (hunk 1 above)
Comment (przemek-sl) [reply docs/basics/sso/configuring-sso-in-okta.md]: fixed

**Review by aoleszyn — APPROVED** [review]

**Pull request merged**